Makefile

```diff
@@ -26,7 +26,13 @@ DOC_FILES := \
 	project.md \
 	media-types.md \
 	manifest.md \
-	serialization.md
+	descriptor.md \
+	image-layout.md \
+	serialization.md \
+	layer.md \
+	GOVERNANCE.md \
+	RELEASES.md
```

Comment:
Do we need these last two?  I think they're metadata, and not part of the spec itself.  runtime-spec [does not include them in the generated spec][1] or link to them from any files which are part of the generated spec.

[1]: https://github.com/opencontainers/runtime-spec/blob/22c29c18eddc75ccee9f6b202f61524f29653c5f/Makefile#L23-L38